src/modules/decklink/consumer/decklink_consumer.cpp

```diff
@@ -225,6 +225,8 @@ struct HDRMetadata
 const auto REC_709  = ChromaticityCoordinates{0.640, 0.330, 0.300, 0.600, 0.150, 0.060, 0.3127, 0.3290};
 const auto REC_2020 = ChromaticityCoordinates{0.708, 0.292, 0.170, 0.797, 0.131, 0.046, 0.3127, 0.3290};
 
+typedef const char* BSTR;
```

Comment:
Theres already an typedef called `String` inside decklink_api.h which handles this abstraction

Reply:
Ah, that's great! Thank you